File: C++/506.cpp
#include <iostream>

using namespace std;

// 506 - Tensión Descompensada - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for (int i = 0; i<loop; i++) {
        int a = 0;
        char division = ' ';
        int b = 0;
        cin >> a;
        cin >> division;
        cin >> b;
        if (a < b) {
            cout << "MAL" << endl;
        } else {
            cout << "BIEN" << endl;
        }
    }
}

File: Java/506.java
import java.util.Scanner;

/**
 *
 * 506 - Tensión Descompensada - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int prueba = sc.nextInt();
        for (int i = 1; i <= prueba; i++) {
            int valor = sc.nextInt();
            char division = sc.next().charAt(0);
            int valor2 = sc.nextInt();

            if (valor >= valor2) {
                System.out.println("BIEN");
            } else {
                System.out.println("MAL");
            }
        }

    }
}
